pkg/payload/precondition/clusterversion/recommendedupdate.go

```diff
@@ -104,13 +104,15 @@ func (ru *RecommendedUpdate) Run(ctx context.Context, releaseContext preconditio
 	} else {
 		msg = fmt.Sprintf("%s=%s (%s), so the recommended status of updating from %s to %s is unknown.", retrieved.Type,
 			retrieved.Status, retrieved.Reason, clusterVersion.Status.Desired.Version, releaseContext.DesiredVersion)
+		warning = true
```

Comment:
Should any of these `RecommendedUpdate` cases should be blocking?  Because any change we make seems like it could break somebody.